src/components/AppMap.ts

```diff
@@ -834,9 +836,50 @@ export default class AppMap extends mixins(MixinUtil) {
       marker.data.getMarker().addTo(this.map.m);
     }
 
+    this.updateTooltip();
     this.searching = false;
   }
 
+  yTooltipOnEach(marker: any) {
+    let m: any = marker.getMarker();
+    if (!('_tooltip' in marker.obj)) {
+      // @ts-ignore
+      let tt = m.getTooltip();
+      marker.obj._tooltip = tt.getContent();
+      marker.obj._tooltip_options = tt.options;
+    }
+    m.unbindTooltip();
```

Comment:
Can you add a comment explaining why this is necessary? (feel free to copy part of your PR description)